advanced/Scripts/chronometer.sh

```diff
@@ -463,8 +463,7 @@ chronoFunc() {
         # Get refresh number
         if [[ "$*" == *"-r"* ]]; then
             num="$*"
-            num="${num/*-r /}"
-            num="${num/ */}"
+            num="${num/*-r* /}"
```

Comment:
Will this equally allow for something like `--railway`?

Reply:
I believe so. The argument parsing needs work overall, and should probably be done in a separate PR. Previously it would have also recognized that argument as a refresh argument.

Reply:
I'm happy with this as is for a quick fix., will approve!

But I guess we should be doing it more like this? https://github.com/pi-hole/pi-hole/blob/master/pihole#L446

Reply:
The issue here seems to be that we allow many different flags, and have a hard time extracting the associated value from the arguments. `pihole` only takes in one sub-command, so it has an easier time. It may be that chronometer should also only take in one command, but it's written to allow multiple.

Reply:
I'll approve as well, but I'll go on record that this is a very bad way of handling flags.

Reply:
It looks like chronometer actually will prevent those other invalid flags eariler:
```
$ pihole -c --railway 1
Unknown option. Please view 'pihole -c --help' for more information
```

I've pushed a commit to make this explicit here, though.